SDKTest/EventTests.swift

```diff
@@ -353,4 +361,90 @@ class EventTests: XCTestCase {
         }
         assert(neuroHTTPRequest != nil)
     }
+    
+    // Specific UI Class Registration Tests
+    func testUITextFieldRegistration() {
+        NeuroID.start()
```

Comment:
Might be overkill but I figured we should test that we are able to register all the UI elements we think we can. These tests will check not only that a `REGISTER_TARGET` event was captured but also that the event type is what we are expecting.